js/interface.js

```diff
@@ -539,8 +541,10 @@ function openApp(appId) {
     });
 }
 
-function openOrganization(organizationId) {
+function openOrganization (organizationId) {
```

Comment:
Looks like ESLint isn't correctly set up here.

```suggestion
function openOrganization(organizationId) {
```